common/utils/create_directory.py

```diff
@@ -0,0 +1,14 @@
+import os
+import logging
+from common.customized_logging import configure_logging
+
+configure_logging()
+logger = logging.getLogger(__name__)
+
+
+def create_directory(dir_path):
+    try:
+        os.mkdir(dir_path)
+        logger.info(f"{dir_path} is created")
+    except OSError:
+        logger.info(f"Skipping creating {dir_path} since it exists.")
```

Comment:
shouldn't be this as a module. basically, you created a module for just one function.  module >> class >> function. move this to an _utils.py_ module somewhere.